### PR TITLE
Resolved Issue: 5

### PR DESCRIPTION
**Pull request by adtoria**
Issue: 5

#### Short description of what this resolves:
- Completed _getTimerText function to make & return a string that displays the remaining time.

#### Changes proposed in this pull request and/or Screenshots of changes:
- NA

### CODE REVIEW

File: gnome/extension.js [hunk 1]
```diff
@@ -117,11 +117,26 @@ const ContestCountdown = new Lang.Class({
...
+      timerText = days + "d " + hours + "h " + minutes + "m ";
+    } else {
+      if (timeDiff == -1) {
+        timerText = "Loading data.";
+      } else {
+        timerText = "No upcoming contests.";
+      } 
+    }
 
-  _refreshUI: function (data) {
```
Comment (rag-hav) [inline gnome/extension.js:122]: dont delete this function

File: gnome/extension.js [hunk 2]
```diff
@@ -117,11 +117,26 @@ const ContestCountdown = new Lang.Class({
   // TODO: Issue #4
   // Make the string to display in top bar using contests objects
   // @return: string
-  _getTimerText: function () {},
+  _getTimerText: function () {
+    var timerText;
+
+    let timeDiff = this.contests.secondsTillNextContest();
+
+    if (timeDiff >= 0) {
```
Comment (rag-hav) [inline gnome/extension.js:125]: +infinity will be handled by this if.

Note: If you find the edge cases given by secondsTillNextContest unwieldy. You can edit them to -1 -2 -3.. or anything. Just make sure to document them properly.

**Review by rag-hav — CHANGES_REQUESTED** [review]
Need to handle edge cases properly

**Comment by adtoria** [comment]
> Need to handle edge cases properly

Made changes. Please review.

File: gnome/extension.js [hunk 3]
```diff
@@ -117,13 +117,34 @@ const ContestCountdown = new Lang.Class({
...
+      } else {
+        var days = Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+        var hours = Math.floor((timeDiff % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+        var minutes = Math.floor((timeDiff % (1000 * 60 * 60)) / (1000 * 60));
+
+        timerText = days + "d " + hours + "h " + minutes + "m ";
+      }    
+    } else {
+      if (timeDiff == -1) {
+        timerText = "Loading data.";
```
Comment (rag-hav) [inline gnome/extension.js:135]: either remove the dot or make it three dots

**Review by rag-hav — CHANGES_REQUESTED** [review]

**Pull request merged**